osu.Game/Database/StandardisedScoreMigrationTools.cs

```diff
@@ -264,9 +266,63 @@ public static long ConvertFromLegacyTotalScore(ScoreInfo score, LegacyBeatmapCon
             switch (score.Ruleset.OnlineID)
             {
                 case 0:
+                    if (score.MaxCombo == 0 || score.Accuracy == 0)
+                    {
+                        return (long)Math.Round((
+                            0
+                            + 300000 * Math.Pow(score.Accuracy, 8)
+                            + bonusProportion) * modMultiplier);
+                    }
+
+                    // Assumption :
+                    // - sliders and slider-ticks are uniformly spread arround the beatmap
+                    //      thus we can ignore them without losing much precision (consider a map of hit-circles only !)
+                    // - the Ok/Meh hit results are uniformly spread in the score
+                    //      thus we can simplify and consider each hit result to be score.Accuracy without losing much precision
+                    // What is strippedV1/strippedV3 :
+                    // This is the ComboScore of v1/v3 were we remove all (map-)constant multipliers and accuracy multipliers (including hit results),
```

Comment:
What is "v3"? Can we not use that term?

Reply:
Yes mybad. (This is from before I was given that instruction)
I changed to new, but it might be better to find a fitting name ?